src/zenml/integrations/seldon/services/seldon_deployment.py

```diff
@@ -46,23 +46,32 @@ class SeldonDeploymentConfig(ServiceConfig):
             should use the same model name.
         implementation: the Seldon Core implementation used to serve the model.
         replicas: number of replicas to use for the prediction service.
+        parameters: parameters to pass to the Seldon Core implementation.
         secret_name: the name of a Kubernetes secret containing additional
             configuration parameters for the Seldon Core deployment (e.g.
             credentials to access the Artifact Store).
+        container_registry_secret_name: the name of a Kubernetes secret containing
+            additional configuration parameters for the container registry.
         model_metadata: optional model metadata information (see
             https://docs.seldon.io/projects/seldon-core/en/latest/reference/apis/metadata.html).
         extra_args: additional arguments to pass to the Seldon Core deployment
             resource configuration.
+        is_custom_deployment:
+        spec: custom Kubernetes resource specification for the Seldon Core
```

Comment:
```suggestion
        spec: custom Kubernetes resource specification for the Seldon Core.
```

These should all end with a `.`